test/Microsoft.VisualStudio.Composition.Tests/StaticMemberExportsTests.cs

```diff
@@ -194,5 +195,104 @@ public class ImportManyWithMetadataPart
             [MefV1.ImportMany("Property")]
             public List<Lazy<string, IDictionary<string, object>>> ImportingMember { get; set; } = null!;
         }
+
+        // Test class to verify static members don't cause instantiation
+        public class ClassWithStaticMemberExports
+        {
+            private static bool constructorCalled = false;
+
+            public static bool ConstructorCalled
+            {
+                get { return constructorCalled; }
+                set { constructorCalled = value; }
+            }
+
+            public ClassWithStaticMemberExports()
+            {
+                constructorCalled = true;
+            }
+
+            [MefV1.Export("StaticField")]
+            [MefV2.Export("StaticField")]
```

Comment:
Drop the MEFv2 attribute and V2Compat on the test for this member, since MEFv2 does not support exporting fields.